modules/WordCloudPlugin/src/main/java/net/clementlevallois/wordcloud/LexplorerTopComponent.java

```diff
@@ -0,0 +1,551 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/NetBeansModuleDevelopment-files/templateTopComponent637.java to edit this template
+ */
+package net.clementlevallois.wordcloud;
+
+import java.awt.Desktop;
+import java.beans.PropertyVetoException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ResourceBundle;
+import javax.swing.DefaultListModel;
+import javax.swing.ListSelectionModel;
+import javax.swing.SpinnerModel;
+import javax.swing.SpinnerNumberModel;
+import javax.swing.SwingWorker;
+import org.gephi.graph.api.GraphModel;
+import org.gephi.utils.longtask.api.LongTaskExecutor;
+import org.netbeans.api.settings.ConvertAsProperties;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+
+/**
+ * Top component which displays something.
+ */
+@ConvertAsProperties(
+        dtd = "-//net.clementlevallois.wordcloud//Lexplorer//EN",
+        autostore = false
+)
+@TopComponent.Description(
+        preferredID = "LexplorerTopComponent",
+        //iconBase="SET/PATH/TO/ICON/HERE",
+        persistenceType = TopComponent.PERSISTENCE_ALWAYS
+)
+@TopComponent.Registration(mode = "filtersmode", openAtStartup = true)
```

Comment:
I'm not sure it makes sense for plugins to open at startup. I don't have a strong opinion. On one hand it makes it easier to find. On the other hand if every plugin opens new components that ends up not being used during the session it can slow down Gephi.

Reply:
I took a default template for the plugin, that is a default value. Ok I'll turn it to false!

Reply:
Yes, I believe the default is `true`. Maybe we should change that.